core/src/main/java/org/elasticsearch/common/unit/ByteSizeValue.java

```diff
@@ -105,26 +134,33 @@ public double getPbFrac() {
         return ((double) getBytes()) / ByteSizeUnit.C5;
     }
 
+    public String getStringRep() {
```

Comment:
Please add javadocs to explain how this differs from toString()

Reply:
Agreed, see https://github.com/elastic/elasticsearch/pull/27702/commits/ad16a5ce33b4b3b045f67aa5d5471a0ed68597d2